core/base/ftmTree/FTMTree_MT.cpp

```diff
@@ -805,20 +805,8 @@ void FTMTree_MT::sortLeaves(const bool para) {
   };
 
   if(para) {
-#ifdef __clang__
-    std::sort(mt_data_.leaves->begin(), mt_data_.leaves->end(), indirect_sort);
-#else
-#ifndef _MSC_VER
-#ifdef TTK_ENABLE_OPENMP
-    __gnu_parallel::sort(
-      mt_data_.leaves->begin(), mt_data_.leaves->end(), indirect_sort);
-#else
-    std::sort(mt_data_.leaves->begin(), mt_data_.leaves->end(), indirect_sort);
-#endif
-#else
-    std::sort(mt_data_.leaves->begin(), mt_data_.leaves->end(), indirect_sort);
-#endif
-#endif
+    PSORT(this->threadNumber_)
```

Comment:
I'd recommend to rename in the future this macro into `TTK_PSORT`, to make it clear to developers that they can find the definition of the macro within TTK.

Reply:
(see my other comments on slack)